api/v1alpha1/ipfscluster_types.go

```diff
@@ -54,6 +59,14 @@ type ReprovideSettings struct {
 	Interval string `json:"interval,omitempty"`
 }
 
+type ExternalSettings struct {
+	// +kubebuilder:validation:Enum={ingress,loadbalancer,none}
+	// +optional
+	Strategy ExternalStrategy `json:"strategy,omitempty"`
```

Comment:
What exactly are these fields supposed to change or control? Could you please include a one-sentence comment above the new fields and structs being added?